.github/workflows/velox_backend.yml

```diff
@@ -638,6 +638,12 @@ jobs:
         with:
           name: test-report-spark32
           path: '**/surefire-reports/TEST-*.xml'
+      - name: Upload unit tests log files
+        if: ${{ !success() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: unit-tests-log-spark32
```

Comment:
Suggest to simply reference the job name, e.g., `name: "${{ github.job }}-test-log"`. Ditto for other places.

Reply:
This name aligns w/ the test-report* and golden-files*, I don't think we need to prefix it w/ job names?

Reply:
I would recommend to change golden file name also.
It' just for well distinguishing which job produces the file. It's also for better maintenance, especially when we add more test job derived from an existing one. I note `golden-files-spark35` was unexpected both used as golden file name for JDK 8 and 17 test job, which is a small mistake easy to make.

Reply:
OK, as we're not leveraging the matrix feature of GA, it looks like a good idea for avoiding copy-paste issues.